strings/charoccurrence_test.go

```diff
@@ -0,0 +1,39 @@
+package charoccurrence_test
```

Comment:
```suggestion
package strings_test
```
The tests would be failing...

Reply:
- [x] fixed